mods/ra/maps/singles-in-the-rain/map.yaml

```diff
@@ -0,0 +1,170 @@
+MapFormat: 7
```

Comment:
I'd like to see how the weather performs with scrolling at a bigger map. The F.Lonestar funmap is to small for nice radar based viewport jumps. Theres also no ocean at F. Lonestar. And I'd also like to see the droplets over the watersurface and future more maybe there will be these littel raindrop circles on the water surface in the future (Not sure yet). And Fort Lonestar is using a night setting, what would cause me to use darker particle colors.
That's why I think F.Lonestar is not the better map.